README.md

```diff
@@ -58,3 +58,124 @@ HUBSPOT_CAMPAIGN_ID=<campaign id>
 ```
 
 If you do not have access to 1Password, please reach out to one of the maintainers of the project.
+
+---
+
+## Adding a Course
+
+> (This section may have errors and missing information)
+
+Create your course in the `src` folder. For best results, follow the organization and formatting of the Angular course. Course is available by typing in the appropriate url. Update `bit-u.md` to make it appear on the `/academy` page.
```

Comment:
> follow the organization and formatting of the Angular course

You should be able to link to this directory from the readme, could be a nice plus

Reply:
done!